### ISSUE TITLE
Training on custom data and outputs

### ISSUE THREAD

**Issue by rameshshreyas** [Li-Chongyi/Zero-DCE#59]
Hi @Li-Chongyi,

Thanks for this implementation! I have also implemented your paper and trained on my data that consists of images of road at night. The output from ZeroDCE seems to blow up some of the brighter ones as it tries to brighten the darker patches. I used your pretrained model as well as trained on a combination of data from LoL, as well as my vehicle datasets.

![2023-10-17-06-16-22_002675_enhanced](https://github.com/Li-Chongyi/Zero-DCE/assets/70128920/a63db9ef-84bb-41da-8899-bd09cacce2b4)
![2023-10-17-06-16-22_002675_original](https://github.com/Li-Chongyi/Zero-DCE/assets/70128920/ddd38d4e-cc2b-4b9a-a982-c523cbb7486c)

Any pointers to tweak the loss weights would be greatly appreciated.
